frontend/src/components/nonprofitApply/ApplicationProcess.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import { Typography, Box } from '@mui/material'
+
+
+
+
+
+function ApplicationProcess() {
+    return <div>
+
+    
+        <br></br>
+        
+        <Typography variant='h4' align="center" style={{ fontWeight: 'bold' }}>
+            Nonprofits Application Process
+        </Typography>
+
+       <br></br>
+
+       <Box sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            ml: 60,
+            mr: 60
+        }}>
+
+          <Typography marginRight={5} fontSize={80} color={'primary.main'} fontWeight={500}>
+               01
+          </Typography>
+
+          <Typography>
+               <h2>Outreach</h2>
+          
+               <h4 style={{color: '#AFAFAF'}}>Before the start of Fall term (September), we will be conducting outreach to find our yearlong partners!
+                If your nonprofit is interested in an application, free of charge, email us at team@lablueprint.org!</h4>
+          </Typography>
+          </Box>
+       
+          <Typography marginLeft={65} fontSize={50} color={'primary.main'} fontWeight={'bold'}>
+               |
+          </Typography>
+
+       <Box sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            ml: 60,
+            mr: 60
+        }}>
+
+
+          <Typography marginRight={5} fontSize={80} color={'primary.main'} fontWeight={500}>
+               02
+          </Typography>
+
+          <Typography>
+               <h2>Scoping Call</h2>
+          
+               <h4 style={{color: '#AFAFAF'}}>Our team will set up a call to learn more about your nonprofit’s technical goals.
+                We are dedicated to hearing about your organization’s current approach to driving social good and how we might help you.</h4>
+          </Typography>
+          </Box>
+      
+          <Typography marginLeft={65} fontSize={50} color={'primary.main'} fontWeight={'bold'}>
+               |
+          </Typography>
+
+       <Box sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            ml: 60,
+            mr: 60
+        }}>
+
+          <Typography marginRight={5} fontSize={80} color={'primary.main'} fontWeight={500}>
+               03
+          </Typography>
+
+          <Typography>
+               <h2>Nonprofit Project Proposal Application</h2>
+          
+               <h4 style={{color: '#AFAFAF'}}>After our scoping call, we ask you to fill out this project proposal application. 
+               Our team is committed to answering any questions or setting up additional meetings to support you.</h4>
```

Comment:
we also don't want to hardcode the content (title, number, description). Instead we will have a json file in src/data containing array of objects containing each steps and then we will loop through this array in the react file creating a ApplyProcessStep each time (and passing the data as a prop). This might be a bit confusing if you're just starting so let me know if you need help, but you can see an example in components/about/member.js.

Eventually we would replace the local json file with API call to our backend but this is a future concern.